test/functional/feature_rbf.py

```diff
@@ -714,5 +717,33 @@ def test_replacement_relay_fee(self):
         tx.vout[0].nValue -= 1
         assert_raises_rpc_error(-26, "insufficient fee", self.nodes[0].sendrawtransaction, tx.serialize().hex())
 
+    def test_fullrbf(self):
+        txid = self.wallet.send_self_transfer(from_node=self.nodes[0])['txid']
+        self.generate(self.nodes[0], 1)
+        confirmed_utxo = self.wallet.get_utxo(txid=txid)
+
+        self.restart_node(0, extra_args=["-mempoolfullrbf=1"])
+
+        # Create an explictily opt-out transaction
+        optout_tx = self.wallet.send_self_transfer(
+            from_node=self.nodes[0],
+            utxo_to_spend=confirmed_utxo,
+            sequence=SEQUENCE_FINAL,
+            fee_rate=Decimal('0.01'),
+        )
+        assert_equal(False, self.nodes[0].getmempoolentry(optout_tx['txid'])['bip125-replaceable'])
+
+        conflicting_tx = self.wallet.create_self_transfer(
+                utxo_to_spend=confirmed_utxo,
+                sequence=SEQUENCE_FINAL,
+                fee_rate=Decimal('0.02'),
+        )
+
+        # Test if optout_tx can be replaced
+        res = self.nodes[0].testmempoolaccept(rawtxs=[conflicting_tx['hex']])[0]
```

Comment:
is there a real benefit for testmempoolaccept vs submission? Would be nice to explicitly check that the first tx got booted from mempool rather than implicitly

Reply:
Updated with `sendrawtransaction` and testing that `optout_tx` is not in the mempool after.
On a more general note, I think for now it should be equivalent to test with either `sendrawtransaction` / `testmempoolaccept` though they might differ more as we those code paths start to be more special-cased with package/set of transactions validation.